app/sessionStore.js

```diff
@@ -432,6 +432,16 @@ module.exports.loadAppState = () => {
         }
       }
 
+      // sites refactoring migration
+      if (Array.isArray(data.sites) && data.sites.length) {
+        let sites = {}
+        data.sites.forEach((site) => {
+          let key = siteUtil.getSiteKey(Immutable.fromJS(site), site.tags)
+          sites[key] = site
+        })
+        data.sites = sites
+      }
+
```

Comment:
Great job with the above :smile: It's super important to have migrations for the session data